Styles/style.css

```diff
@@ -0,0 +1,976 @@
+@media only screen and (max-width: 767px) {
```

Comment:
Taking into account that we are building following the mobile-first approach regarding the style, this first media query is redundant;
We should build the style from mobile. And for the larger screen, we just adapt the style for the components that need to be changed. In this way, we don't violate the DRY (Don't Repeat Yourself) principle. 
 - [ ] you can take into account to remove this media query.
 - [ ] for the desktop, apply the change only for the elements that need to be changed;